tobac/themes/tobac_v1/feature_detection.py

```diff
@@ -63,6 +64,13 @@ def feature_detection_multithreshold(
         Flag choosing method used for the position of the tracked
         feature. Default is 'center'.
 
+    coord_interp_kind : str, optional
+        The kind of interpolation as a string. Default is 'linear'.
```

Comment:
I think this should be clearer as to what the interpolation is touching- just add something like "for coordinates" in here.